webapp-django/crashstats/crashstats/models.py

```diff
@@ -987,11 +987,19 @@ def get(self, bugs):
             url = settings.BZAPI_BASE_URL + (
                 '/bug?id=%(bugs)s&include_fields=%(fields)s' % params
             )
-            response = requests_retry_session().get(
+            response = requests_retry_session(
+                # BZAPI isn't super reliable, so be extra patient
+                retries=5,
+                # 502 = Bad Gateway
+                # 504 = Gateway Time-out
+                status_forcelist=(500, 502, 504),
+            ).get(
                 url,
                 headers=headers,
                 timeout=self.BUGZILLA_REST_TIMEOUT,
             )
+            # Better fail here than getting a JsonDecodeError on the next line.
+            assert response.status_code == 200, response.status_code
```

Comment:
If, against all odds, you do get an `AssertionError` you'll see where in the code it was and you'll have to look it up and it'll make sense in my humble opinion. 

If BZPI is really really down and keep responding with 500 errors, the lines above will raise a `Retry` error (I think it's `requests.exceptions.Retry` or something)

I suspect @adngdb is going to say "remove the assert anyway" :)

I just thought it was a slight upgrade from getting a `JsonDecodeError` on the next line.